src/types/parameter.d.ts

```diff
@@ -27,6 +28,8 @@ type ParametersErrorMap = Record<ParameterName, ParameterError>;
 
 type ParameterName = string;
 
+type RequiredParametersList = string[];
```

Comment:
Can you use `ParameterName[]` instead of `string[]` here?

Reply:
Sure thing.